### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by hugheslaron**
Jira Ticket: [COV-420](https://ctds-planx.atlassian.net/browse/COV-420)

- Major Harmonization Effort

### New Features
N/A

### Breaking Changes
Yes

### Bug Fixes
N/a

### Improvements
N/A

### Dependency updates
N/A

### Deployment changes
N/A

### CODE REVIEW

File: gdcdictionary/schemas/subject.yaml [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+    description: >
+      NCBI identifier for classification of organisms.
+    type: integer
+    termDef:
+      - term: taxon_id
+        source: NCI Thesaurus
+        term_id: C54539
+        term_version: 20.03e (Release date:2020-03-31)
+
+  genus:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml]: Didn't we decide that we would be removing genus from the subject node?

Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml]: Yes, it was decided to delete it from the Subject node

File: follow_up.yaml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      Yes (Y) or no (N) or blank if unknown
+    enum:
+      - "Y"
+      - "N"
+
+  in_icu:
+    description: >
+      Yes (Y) if the patient is in the ICU (intensive care unit) or CCU (critical care unit) or No (N) or blank if unknown.
+    enum:
+      - "Y"
```
Comment (gkuffel) [inline follow_up.yaml:64]: I think we said we would make all enums True and False.

File: follow_up.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      - "N"
+
+  extubated:
+    description: >
+      Yes (Y) if the patient was successfully extubated or No (N) or blank if unknown
+    enum:
+      - "Y"
+      - "N"
```
Comment (gkuffel) [inline follow_up.yaml:72]: True & False

File: follow_up.yaml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+  temperature:
+    description: >
+      Temperature of the patient in Celsius at the time of the image
+    type: number
+
+  pO2_saturation:
+    description: >
+      partial pressure of oxygen saturation in % at the time of the image
```
Comment (gkuffel) [inline follow_up.yaml:81]: Capitalize the P in partial and add a period at the end of the sentence.

File: follow_up.yaml [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+  view:
+    description: >
+      Posteroanterior (PA), Anteroposterior (AP), AP Supine (APS), or Lateral (L) for X-rays; Axial or Coronal for CT scans
+    type: string
+
+  modality:
+    description: >
+      CT, X-ray, or other
+    type: string
```
Comment (gkuffel) [inline follow_up.yaml:107]: Change description to "Medical imaging modalities, for example CT Scan, X-ray, or MRI.

File: gdcdictionary/schemas/demographic.yaml [hunk 6]
```diff
@@ -288,46 +214,61 @@ properties:
...
+  age:
     description: >
-      The participant's age (in years) on the reference or anchor date used during date obfuscation. If this age is greater than 89 years, see 'human_age_at_index_gt89'.
-    type:
-      - number
-      - "null"
-    maximum: 89
-    minimum: 0
+      Please give age in either: 
+      -years (most common)
```
Comment (gkuffel) [inline gdcdictionary/schemas/demographic.yaml:220]: Capitalize these enums

File: follow_up.yaml (hunk 2 above)
Comment (hugheslaron) [reply follow_up.yaml:64]: Done

File: gdcdictionary/schemas/demographic.yaml [hunk 7]
```diff
@@ -288,46 +214,61 @@ properties:
...
     description: >
-      Indicate if the participant's age (in years) on the reference or anchor date used during date obfuscation is greater than 89 years.
-    enum:
-      - "Yes"
-      - "No"
+      County of residence
+    type: string
 
-  nationality:
+  res_state:
```
Comment (gkuffel) [inline gdcdictionary/schemas/demographic.yaml:243]: We decided that we would be using province_state to capture this information as indicated in the Core Props SS.

File: follow_up.yaml (hunk 3 above)
Comment (hugheslaron) [reply follow_up.yaml:72]: Done

File: follow_up.yaml (hunk 4 above)
Comment (hugheslaron) [reply follow_up.yaml:81]: Done

File: gdcdictionary/schemas/subject.yaml [hunk 8]
```diff
@@ -50,21 +50,6 @@ uniqueKeys:
...
-     - Mustela putorius furo
-     - Rattus rattus
-     - Sus scrofa
-    termDef:
-      - term: species
-        source: NCI Thesaurus
-        term_id: C45293
-        term_version: 20.03e (Release date:2020-03-31)
-
   strain:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:53]: Remove this property.

File: gdcdictionary/schemas/subject.yaml [hunk 9]
```diff
@@ -85,16 +70,6 @@ properties:
...
-    description: >
-      Genus of organism collected.
-    type: string
-    termDef:
-      - term: taxon_id
-        source: NCI Thesaurus
-        term_id: C45292
-        term_version: 20.03e (Release date:2020-03-31)
-
   genome_length:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:73]: Remove this property.

File: follow_up.yaml (hunk 5 above)
Comment (hugheslaron) [reply follow_up.yaml:107]: done

File: gdcdictionary/schemas/demographic.yaml (hunk 6 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/demographic.yaml:220]: done

File: gdcdictionary/schemas/subject.yaml [hunk 10]
```diff
@@ -105,110 +80,23 @@ properties:
...
-    description: >
-      Country
-    type: string
-    termDef:
-      - term: country
-        source: NCI Thesaurus
-        term_id: C25464
-        term_version: 20.03e (Release date:2020-03-31)
-
   wuhan_not_wuhan:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:83]: Remove this property.

File: gdcdictionary/schemas/subject.yaml [hunk 11]
```diff
@@ -105,110 +80,23 @@ properties:
...
-      Longitude
-    type:
-      - string
-    termDef:
-      - term: longitude
-        source: NCI Thesaurus
-        term_id: C68643
-        term_version: 20.03e (Release date:2020-03-31)
-
   geo_resolution:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:90]: Remove this property.

File: gdcdictionary/schemas/subject.yaml [hunk 12]
```diff
@@ -105,110 +80,23 @@ properties:
...
-    type: array
-    items:
-      type: string
-    termDef:
-      - term: symptoms
-        source: NCI Thesaurus
-        term_id: C4876
-        term_version: 20.03e (Release date:2020-03-31)
-
   lives_in_wuhan:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:100]: This prop should be deleted. It is being harmonized with the city property.

File: gdcdictionary/schemas/subject.yaml [hunk 13]
```diff
@@ -257,21 +136,6 @@ properties:
...
-        source: NCI Thesaurus
-        term_id: C25683
-        term_version: 20.03e (Release date:2020-03-31)
-
-  sequence_available:
-    description: >
-      Sequence available
-    type: string
-
   outcome:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:139]: Remove property.

File: gdcdictionary/schemas/subject.yaml [hunk 14]
```diff
@@ -238,15 +126,6 @@ properties:
...
-  chronic_disease_binary:
-    description: >
-      Chronic Disease Binary
-    enum:
-      - "0"
-      - "1"
-      - "NA"
-      - "null"
-
   chronic_disease:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:129]: Remove property.

File: gdcdictionary/schemas/subject.yaml [hunk 15]
```diff
@@ -302,55 +161,13 @@ properties:
...
-    type:
-      - string
-      - "null"
-    termDef:
-      - term: reporting_date
-        source: NCI Thesaurus
-        term_id: C70857
-        term_version: 20.03e (Release date:2020-03-31)
-
   summary:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:171]: Remove property.

File: gdcdictionary/schemas/subject.yaml [hunk 16]
```diff
@@ -463,35 +206,13 @@ properties:
         term_id: C68615
         term_version: 20.03e (Release date:2020-03-31)
 
-  age_decade:
-    description: >
-      the age of the patient in decades
-    type: string
-
   isolated:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:209]: This property was recommended for removal and will be captured instead by the isolation_status property.

File: gdcdictionary/schemas/subject.yaml [hunk 17]
```diff
@@ -463,35 +206,13 @@ properties:
...
-    enum:
-      - "True"
-      - "False"
-
-  infection_case:
-    description: >
-      the case of infection
-    type: string
-
   infection_order:
```
Comment (gkuffel) [inline gdcdictionary/schemas/subject.yaml:216]: Remove property.

File: gdcdictionary/schemas/subject.yaml (hunk 8 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:53]: done

File: gdcdictionary/schemas/subject.yaml (hunk 9 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:73]: done

File: gdcdictionary/schemas/subject.yaml (hunk 10 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:83]: done

File: gdcdictionary/schemas/subject.yaml (hunk 11 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:90]: done

File: gdcdictionary/schemas/subject.yaml (hunk 12 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:100]: done

File: gdcdictionary/schemas/subject.yaml (hunk 13 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:139]: done

File: gdcdictionary/schemas/subject.yaml (hunk 14 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:129]: done

File: gdcdictionary/schemas/subject.yaml (hunk 15 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:171]: done

File: gdcdictionary/schemas/subject.yaml (hunk 16 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:129]: done

File: gdcdictionary/schemas/subject.yaml (hunk 17 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/subject.yaml:136]: done

File: gdcdictionary/schemas/demographic.yaml (hunk 6 above)
Comment (hugheslaron) [reply gdcdictionary/schemas/demographic.yaml:220]: done

File: gdcdictionary/schemas/demographic.yaml [hunk 18]
```diff
@@ -288,46 +214,56 @@ properties:
...
-      - term: Human Age At Index
+      - term: ageunit
         source: NCI Thesaurus
-        term_id: C25390
-        term_version: 18.12e (Release date:2018-12-31)
+        term_id: C50400
+        term_version: 20.03e (Release date:2020-03-31)
 
-  human_age_at_index_gt89:
+  res_county:
```
Comment (gkuffel) [inline gdcdictionary/schemas/demographic.yaml:238]: Let's change this prop name to just county to be consistent with the Summary Location node.

**Pull request merged**